tests/puppeteer/campaigns/utils/initPages.js

```diff
@@ -0,0 +1,47 @@
+require('module-alias/register');
+const BOBasePage = require('@pages/BO/BObasePage');
+const FOBasePage = require('@pages/FO/FObasePage');
+const LoginPage = require('@pages/BO/login');
+const DashboardPage = require('@pages/BO/dashboard');
+const HomePage = require('@pages/FO/home');
+const InstallPage = require('@pages/Install/install');
+
+module.exports =  async function initBasicPages(context = 'BO', pagesObjects, page) {
+  let baseObject;
```

Comment:
```suggestion
  let baseObject;
  
```

I love to get some spaces in my code, it increase readability as hell, before boucles, conditions...